### ISSUE TITLE
Add session user to audit log

### ISSUE THREAD

**Issue by anusharanganathan** [culturesofknowledge/emlo-project#27]

**Comment by philipkcl** [comment]
@anusharanganathan  to be discuss this one

**Comment by J4bbi** [comment]
@philipkcl , this is done?

**Comment by philipkcl** [comment]
@J4bbi, I need more information for this one, do you have any idea ?

**Comment by J4bbi** [comment]
The problem this issue fails to describe is that in the old system the Postgres user would be logged as the author of a change. In our system we want to write the Django user into the audit log. So the problem is, how to reconcile these two, different concepts of users.

If I remember correctly, we will still keep all the database audit triggers, and if so, does a signal on the audit table intercept the new entry and replace the Postgres user with the Django username?

**Comment by philipkcl** [comment]
your remember is correct,

but how important to "reconcile these two"?

the username field in audit should be for display only, no other features right?

**Comment by J4bbi** [comment]
> the username field in audit should be for display only, no other features right?
I'm not sure, I think so

**Comment by philipkcl** [comment]
system seem showing ''__unknown_user" after using uploader. to be study that problem

**Comment by anusharanganathan** [comment]
@J4bbi is there anything more needed to be done here? Rama recently worked on issue https://github.com/culturesofknowledge/emlo-project/issues/711.

**Comment by mirandalewis** [comment]
We're not sure if we can close it, but as far as I can tell in our interface it's still open:

<img width="314" height="194" alt="Image" src="https://github.com/user-attachments/assets/979fbb0f-58bd-4e3d-b7e6-88f0e59496a5" />

**Comment by MillieG711** [comment]
@anusharanganathan is there any difference between this and #711?

**Comment by anusharanganathan** [comment]
> [@anusharanganathan](https://github.com/anusharanganathan) is there any difference between this and [#711](https://github.com/culturesofknowledge/emlo-project/issues/711)?

@mirandalewis and @MillieG711 this is one for @J4bbi to check. He will look and let us know if it can be closed.

**Comment by J4bbi** [comment]
Fixed by https://github.com/culturesofknowledge/site-edit-2/pull/205

**Comment by anusharanganathan** [comment]
We will keep this open until we do a fresh import and check the __unknown_user__ issue has gone. The session user is being added to the audit logs as tested in other issues.

**Comment by MillieG711** [comment]
@anusharanganathan can we close this now that the import has been done?

**Comment by anusharanganathan** [comment]
@MillieG711 yes, we can close this one, now that we no longer see unknown user ([example audit search](https://emlo-edit.cottagelabs.com/audit/search?recref_mode=0&change_timestamp_from=01%2F06%2F2024&change_timestamp_to=30%2F06%2F2024&table_name=&key_value_integer_lookup=equals&key_value_integer=&key_decode_lookup=contains&key_decode=&column_name=&change_type=&change_made_lookup=contains&change_made=&audit_id_lookup=equals&audit_id=&order=desc&sort_by=change_timestamp&num_record=100&page=))
